packages/core/src/command/consolidated/consolidated-create.ts

```diff
@@ -35,20 +35,72 @@ export async function createConsolidatedFromConversions({
   patientId,
   destinationBucketName = getConsolidatedLocation(),
   sourceBucketName = getConsolidatedSourceLocation(),
-  logMemUsage,
 }: ConsolidatePatientDataCommand): Promise<Bundle> {
   const { log } = out(`createConsolidatedFromConversions - cx ${cxId}, pat ${patientId}`);
 
-  const consolidated = buildConsolidatedBundle();
+  const [conversions, docRefs] = await Promise.all([
+    getConversions({ cxId, patientId, sourceBucketName }),
+    getDocumentReferences({ cxId, patientId }),
+  ]);
+  log(`Got ${conversions} resources from conversions`);
 
-  const inputBundles = await listConversionBundles({ cxId, patientId, sourceBucketName, log });
-  if (!inputBundles || inputBundles.length < 1) {
+  const withDups = buildConsolidatedBundle();
+  withDups.entry = [...conversions, ...docRefs.map(buildBundleEntry)];
+  withDups.total = withDups.entry.length;
+  log(`Added ${docRefs.length} docRefs, to a total of ${withDups.entry.length} entries`);
+
+  log(`Deduplicating consolidated bundle...`);
+  const deduped = deduplicate({ cxId, patientId, bundle: withDups });
+  log(`...done, from ${withDups.entry?.length} to ${deduped.entry?.length} resources`);
+
+  const dedupDestFileName = createConsolidatedDataFilePath(cxId, patientId, true);
```

Comment:
I'd probably move this args to props so it's clear what the boolean controls but not necessary.

Reply:
Will leave as is for now and add as a note for the next PR

Reply:
https://www.notion.so/metriport/Consolidated-from-S3-295444b99058459bad006e1cc2986364?pvs=4#10bb51bb90408046a6e1d102e73a2acf